Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use the official Ruby image as a base image
+FROM ruby:3.3
```

Comment:
Can this import from our `.ruby-version` somehow? The file is brand new and they're already out of sync. :)